jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/MetricAssertions.java

```diff
@@ -27,7 +27,7 @@ static void assertGauge(Metric metric, String name, String description, String u
     assertThat(metric.getName()).isEqualTo(name);
     assertThat(metric.getDescription()).isEqualTo(description);
     assertThat(metric.getUnit()).isEqualTo(unit);
-    assertThat(metric.hasGauge()).isTrue();
+    assertThat(metric.hasGauge()).withFailMessage("Invalid metric type").isTrue();
```

Comment:
[minor] assertion description us usually only used trying to understand why things fail, so rephrasing it like `metric with gauge expected` allows to understand what is missing/wrong without even looking at the assertion code, so things are easier to understand from the failed test output.